services/frontend-service/src/ui/ActionsCart/CheckoutDialog.tsx

```diff
@@ -120,27 +126,26 @@ export const SyncWindowsWarning: VFC<{
 };
 
 export const CheckoutCart: VFC<{ overview: GetOverviewResponse }> = ({ overview }) => {
-    const [openNotify, setOpenNotify] = React.useState(false);
-    const [dialogOpen, setDialogOpen] = React.useState(false);
-    const [notifyMessage, setNotifyMessage] = React.useState('');
-    const { actions } = useContext(ActionsCartContext);
+    const [notify, setNotify] = useState({ open: false, message: '' });
+    const [dialogOpen, setDialogOpen] = useState(false);
+    const { actions, setActions } = useContext(ActionsCartContext);
+    const [lockMessage, setLockMessage] = useState('');
+
+    const updateLockMessage = useCallback((e) => setLockMessage(e.target.value), [setLockMessage]);
 
     const openNotification = useCallback(
         (msg: string) => {
-            setNotifyMessage(msg);
-            setOpenNotify(true);
+            setNotify({ open: true, message: msg });
         },
-        [setOpenNotify, setNotifyMessage]
+        [setNotify]
     );
 
     const closeNotification = useCallback(
         (event: React.SyntheticEvent | React.MouseEvent, reason?: string) => {
-            if (reason === 'clickaway') {
-                return;
-            }
-            setOpenNotify(false);
+            if (reason === 'clickaway') return;
```

Comment:
not necessary in this commit, but in the future, i'd like to have a linter that makes sure every block is wrapped in `{]`, even if it's just one line